packages/analytics-js/src/components/capabilitiesManager/detection/dom.ts

```diff
@@ -30,6 +30,8 @@ const legacyJSEngineRequiredPolyfills: Record<string, () => boolean> = {
   requestAnimationFrame: () => !isFunction(globalThis.requestAnimationFrame),
   cancelAnimationFrame: () => !isFunction(globalThis.cancelAnimationFrame),
```

Comment:
Similar to the other comment, for example `requestAnimationFrame`, `URLSearchParams` are not supported anymore.
We need to check the supported polyfill groups like `default`, `es6`, `dom`, etc if they can fill these gaps for us.

CC @bardisg

Reply:
Moreover, with polyfill groups, the URL also becomes more straightforward for us.

Reply:
Let's use `https://caniuse.com/` for checking the support. https://developer.mozilla.org/en-US/ seems a bit unreliable for some features.

Reply:
Will create a different ticket for that. Let's not block this one.